.env

```diff
@@ -1,4 +1,6 @@
 PORT=4004
-DB_URL=rego.db
+DB_DRIVER="postgresql"
+# DB_SQLITE_URL=rego.db
+DB_POSTGRES_DSN="host=localhost user=postgres password=postgres dbname=postgres port=5432 sslmode=disable"
```

Comment:
hmm why so?

Reply:
Why use dsn and not split it to multiple variables?
This is what gorm gets and I was a bit lazy yesterday to split it myself. But I'll do that now